cdragontoolbox/hashes.py

```
@@ -213,14 +213,17 @@ def substitute_basenames(self):
         for name in progress_iterator(sorted(names)):
             self.check_iter(f"{dir}/{name}" for dir in dirs)
 
-    def _substitute_basename_words(self, paths, words, amount=1):
-        """Substitutes {amount} side by side words in all basenames
-        of all given paths (default=1 for simple 1 word substitution)."""
+    def _substitute_basename_words(self, paths, words, x=1, y=1):
+        """Replaces x side by side words with y words in all basenames of all given paths
+        (default=1 for simple 1 word substitution). x > 0; y > 0 is required."""
 
+        if not (x and y):
+            print("Either x or y is 0.")
+            return
```

If arguments are invalid, raise a ValueError instead of printing.